src/abstractdataframe/abstractdataframe.jl

```diff
@@ -288,38 +288,33 @@ Base.isempty(df::AbstractDataFrame) = size(df, 1) == 0 || size(df, 2) == 0
 ##
 ##############################################################################
 
-head(df::AbstractDataFrame, r::Int) = df[1:min(r,nrow(df)), :]
-head(df::AbstractDataFrame) = head(df, 6)
-tail(df::AbstractDataFrame, r::Int) = df[max(1,nrow(df)-r+1):nrow(df), :]
-tail(df::AbstractDataFrame) = tail(df, 6)
-
 """
-Show the first or last part of an AbstractDataFrame
+    first(df::AbstractDataFrame)
 
-```julia
-head(df::AbstractDataFrame, r::Int = 6)
-tail(df::AbstractDataFrame, r::Int = 6)
-```
-
-**Arguments**
+Get the first row of `df` as a `DataFrameRow`.
+"""
+Base.first(df::AbstractDataFrame) = df[1, :]
```

Comment:
This will throw an error on 0-row data frame. Maybe some better error message would be in place. Note that `first(df, 1)` will work on `0`-row data frame. Same with `last`.

Reply:
I've followed what happens with vectors. I think that makes sense. The error message isn't too bad, it indicates you tried to access row 1 in a 0-row data frame.